shm/Player.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <algorithm>
+#include <memory>
+#include <numeric>
+#include <string>
+#include <vector>
+
+class Player {
+public:
+    Player(std::unique_ptr<Ship> ship, size_t money) {
+        ship_ = std::move(ship);
+        money_ = money;
+        avaliableSpace_ = countAvailableSpace();
+    };
+
+    size_t getMoney() const { return money_; };
+    size_t getAvaliableSpace() const { return avaliableSpace_; };
+    size_t getSpeed() const { return ship_->getSpeed(); };
```

Comment:
I think it should return const value

Reply:
You mean "return const ship_->getSpeed();"

Reply:
I mean `const size_t getMoney()...` etc :)

Reply:
Hmm, I thought that "const" after name of getter do it.

Reply:
const after the function name "promises" that the getter itself won't modify variables.

Reply:
In this case we return copy of values, so I think it isn't necessary to return const, but maybe I'm wrong.

Reply:
On livestream they did not mention about const before get(), so I would stick with this

Reply:
I think that for now it is enough if we have `type functionName() const {}` to ensure that our getters won't modify the variables. 
+1 to @LogarithmDerivative